src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -342,7 +342,8 @@ DistributedTableList(void)
 	Oid relationId = InvalidOid;
 	foreach_oid(relationId, distTableOidList)
 	{
-		DistTableCacheEntry *cacheEntry = DistributedTableCacheEntry(relationId);
+		DistTableCacheEntry *cacheEntry = CitusTableCacheEntry(relationId);
+		Assert(cacheEntry->isCitusTable);
```

Comment:
This assert is the only change which isn't renaming. It asserts this function is consistent with advertised "valid distributed table cache entries"